Jenkinsfile

```diff
@@ -0,0 +1,254 @@
+#!groovy
```

Comment:
Do we want to pull the Jenkinsfile in this PR? It's ok for me, but I hope it does not conflict with some other branches.